src/rebar_prv_shell.erl

```diff
@@ -331,8 +331,8 @@ reread_config(State) ->
     case find_config(State) of
         no_config ->
             ok;
-        ConfigList ->
-            _ = rebar_utils:reread_config(ConfigList),
+        Config ->
+            _ = rebar_utils:reread_config([Config]),
```

Comment:
Why is this required? That sounds like a totally out of band change for this and as far as I can tell the shell was working fine. `find_config` was supposed to return nested lists already; why is this changing now in a CT-related changeset?